test_data/get_playlist.txt

```diff
@@ -0,0 +1,17856 @@
+{
```

Comment:
Is it possible to use a smaller playlist for this test data ?

Reply:
Sure thing
just pushed with a smaller one (~500 lines, without the extra line breaks in the `available_countries` field)